lib/active_model_serializers/adapter/json_api/relationship.rb

```diff
@@ -34,6 +34,11 @@ def as_json
 
         private
 
+        def ensure_valid!(json)
```

Comment:
@NullVoxPopuli I'd have to crack it open again for specifics, but my point earlier was that the jsonapi gem was already behaving correctly - blowing up when this value was `{}`. Might have been an error post-validation, though.